src/Route/RouteDefinition.php

```diff
@@ -44,13 +50,15 @@ public function __construct(
         Route $route,
         array $middlewares,
         array $middlewareGroups,
-        ?string $name
+        ?string $name,
+        bool $ignoreApiVersionMiddlewareGroup
```

Comment:
[m] Tady ti uteklo pojmenování proměnné
```suggestion
        bool $ignoreVersionMiddlewareGroup
```